conandata.yml

```diff
@@ -0,0 +1,14 @@
+sources:
```

Comment:
IT won't work. You need to commit 1 version per branch. Bincrafters don't the same centralized approach like Conan Center Index.

Reply:
I wondered about that.

What's the issue that keeps us from using this package recipe in the conan-center-index?  
As I understand it, this one is the only one that can be used for cross compilation, and to run the tools.

Reply:
Better answering this late than newer. The recipe in CCI did lack the executables that `icu_installer/<version>@bincrafters/stable` has. See also https://github.com/conan-io/conan-center-index/issues/1136

Now, this is not anymore the case and I'm going to archive this in favour of the CCI recipe. Please migrate to it and report any further issues there. Thanks!